.github/workflows/scoped-test.yaml

```diff
@@ -59,6 +59,9 @@ jobs:
             ./.tools
           key: go-cache-${{ runner.os }}-${{ hashFiles('**/go.sum') }}
 
+      - name: Build test tools
+        run: make "$(${PWD} -replace '\\', '/')/.tools/gotestsum"
```

Comment:
This is not portable, but, right now the action runs only on Windows.

Reply:
In principle this could be conditional on the cache not being available, but, I'm afraid if the cache is saved via e2-to-e2-windows action it won't have the tools.